AdyenComponents/OnlineBanking/OnlineBankingComponent.swift

```diff
@@ -0,0 +1,142 @@
+    /// The terms and condition message item.
+    internal lazy var termsAndConditionsLabelItem: FormAttributedLabelItem = .init(originalText:
+                        localizedString(.onlineBankingTermsAndConditions,
+                        configuration.localizationParameters),
+                        link: termsAndConditionsLink,
+                        style: configuration.style.footnoteLabel,
+                        linkTextStyle: configuration.style.linkTextLabel,
+                        identifier: ViewIdentifierBuilder.build(scopeInstance: self,
+                        postfix: ViewIdentifier.termsAndConditionsLabelItem))
+
+    /// The continue button item.
+    internal lazy var continueButton: FormButtonItem = {
+        let item = FormButtonItem(style: configuration.style.mainButtonItem)
+        item.identifier = ViewIdentifierBuilder.build(scopeInstance: self,
+                                                      postfix: ViewIdentifier.continueButtonItem)
+        item.title = localizedString(.continueTitle, configuration.localizationParameters)
+        item.buttonSelectionHandler = { [weak self] in
+            self?.didSelectContinueButton()
+        }
+        return item
+    }()
+
+    /// The Issuer List item.
+    internal lazy var issuerListItem: FormIssuersPickerItem = {
+        let issuerListPickerItem: [IssuerPickerItem] = onlineBankingPaymentMethod.issuers.map({
+            IssuerPickerItem(identifier: $0.identifier, element: $0)
+        })
+        AdyenAssertion.assert(message: "Issuer list should not be empty", condition: issuerListPickerItem.count <= 0)
+
+        let issuerPickerItem = FormIssuersPickerItem(preselectedValue: issuerListPickerItem[0],
+                                        selectableValues: issuerListPickerItem,
+                                        style: configuration.style.textField)
+        issuerPickerItem.title = localizedString(.selectFieldTitle, configuration.localizationParameters)
+        issuerPickerItem.identifier = ViewIdentifierBuilder.build(scopeInstance: self,
+                                                                  postfix: ViewIdentifier.issuerListItem)
+        observe(issuerPickerItem.publisher) { issuerPickerITem in
+            self.selectedIssuer = issuerPickerITem.element
+        }
+        return issuerPickerItem
+    }()
+
+    /// Initializes the Online Banking component.
+    ///
+    /// - Parameter paymentMethod: The Online Banking payment method.
+    /// - Parameter context: The context object for this component.
+    /// - Parameter configuration: The configuration for the component.
+    public init(paymentMethod: OnlineBankingPaymentMethod,
+                context: AdyenContext,
+                configuration: Configuration = .init()) {
+        self.onlineBankingPaymentMethod = paymentMethod
+        self.context = context
+        self.configuration = configuration
+        self.selectedIssuer = onlineBankingPaymentMethod.issuers[0]
```

Comment:
bounds check.
Would be also nice to confirm from backend if this array can never be empty. If it can, I would go with guard break and optional `selectedIssuer` etc.
If I cannot, then assertion would be fine